dev/release.py

```diff
@@ -47,22 +56,33 @@ def main(release_version, next_version, publish_to, no_prompt):
         sys.exit(1)
 
     current_branch = check_output(["git", "rev-parse", "--abbrev-ref", "HEAD"]).strip()
-    if current_branch != "master":
+    if current_branch != b"master":
         if not verify("You're not on the master branch do you want to continue? (y/n)",
                       interactive):
             sys.exit(1)
 
     uncommitted_changes = check_output(["git", "diff", "--stat"])
-    if uncommitted_changes != "":
+    if uncommitted_changes != b"":
         print(uncommitted_changes)
         print("There seem to be uncommitted changes on your current branch. Please commit or "
               "stash them and try again.")
         sys.exit(1)
 
-    working_branch = WORKING_BRANCH % release_version
-    if working_branch in check_output(["git", "branch"]):
-        prominentPrint(
-            "Working branch %s already exists, please delete it and try again." % working_branch)
+    working_branch = WORKING_BRANCH % (release_version, time.hour, time.minute, time.second)
```

Comment:
It is missing date. It might be easier convert the timestamp into a single string first and include it in the template here.

Reply:
We need to use a custom format for the datetime because the default has characters that cannot be used in a git branch name, I'll update the custom format to include the date.